### ISSUE TITLE
Data Preparation

### ISSUE THREAD

**Issue by gexiaosulynn** [MaybeShewill-CV/lanenet-lane-detection#9]
Thanks a lot for sharing your code. I am confused that both the lanenet and h-net have the same groundtruth  from the paper,but the readme file on github said that I need to organize my training data refer to the data/training_data_example folder structure.I want to know when I train my own modle on tusimple dataset which has json file as label data, if i need to prepare binary segmentation and instance segmentation images as groundtruth  for training, and how to deal the images to obtain instance label?

**Comment by MaybeShewill-CV** [comment]
@gexiaosulynn Each different lane instance should be construct with different pixel value
